lib/activerecord-import/adapters/abstract_adapter.rb

```diff
@@ -45,7 +45,7 @@ def post_sql_statements( table_name, options ) # :nodoc:
       post_sql_statements = []
 
       if supports_on_duplicate_key_update? && options[:on_duplicate_key_update]
-        post_sql_statements << sql_for_on_duplicate_key_update( table_name, options[:on_duplicate_key_update], options[:primary_key] )
+        post_sql_statements << sql_for_on_duplicate_key_update( table_name, options[:locking_column], options[:on_duplicate_key_update], options[:primary_key] )
```

Comment:
Let's make this:

`sql_for_on_duplicate_key_update( table_name, options[:on_duplicate_key_update], options[:primary_key], options[:locking_column])`